src/eliminate_contiguous.cpp

```diff
@@ -9,19 +9,55 @@
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
-bool try_compute_shape(const operation& op, const std::vector<instruction_ref>& args)
+static bool try_compute_shape(instruction_ref ins, const std::vector<shape>& inputs)
 {
     try
     {
-        compute_shape(op, args);
+        shape new_shape = ins->get_operator().compute_shape(inputs);
+        // If the output shape is a standard shape, no need to try its output
+        if(new_shape.standard())
```

Comment:
Also if `new_shape == ins->get_shape()` then we dont need to check the outputs either.

Reply:
If the two shapes are the same, we still cannot decide whether the contiguous can be removed, so we should continue computing later operators to make the decision.

Reply:
> If the two shapes are the same, we still cannot decide whether the contiguous can be removed,

Why? If the shapes are the same than any later shape computation from the outputs will be the same.

Reply:
fixed